microservices/uyuni-java-parent/uyuni-java-parent.spec

```
@@ -0,0 +1,46 @@
+#
+# spec file for package coco-attestation
+#
+# Copyright (c) 2024 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+Name:           uyuni-java-parent
+Version:        1.0.0
```

```suggestion
Version:        5.0.0
```
please, let's align the versions  already